scalameta/semanticdb-scalac/src/test/scala/scala/meta/tests/SemanticSuite.scala

```diff
@@ -3,6 +3,18 @@ package scala.meta.tests
 import scala.meta._
 import scala.meta.internal.semanticdb.SemanticdbMode
 
+// Contributing tips:
```

Comment:
@fikovnik Anything else you can recommend for future contributors? Or any other question you'd like to see addressed here? :)

Reply:
You guys actually did really nice set up. I used did exactly that - started with a simple copy of `SemanticSuite` (added a `println` of the obtained variable in `synthetics`) and ran sbt with `-jvmdebug` with attached debuggr. It was very smooth!

Sorry, I did not realized the alphabetic naming of the test objects :-|